feat(airflowctl): add first release notes and config.toml for it (#54683)
mistercrunch/Airflow: airflow-ctl/RELEASE_NOTES.rst

 .. Licensed to the Apache Software Foundation (ASF) under one
    or more contributor license agreements.  See the NOTICE file
    distributed with this work for additional information
    regarding copyright ownership.  The ASF licenses this file
    to you under the Apache License, Version 2.0 (the
    "License"); you may not use this file except in compliance
    with the License.  You may obtain a copy of the License at

 ..   http://www.apache.org/licenses/LICENSE-2.0

 .. Unless required by applicable law or agreed to in writing,
    software distributed under the License is distributed on an
    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
    KIND, either express or implied.  See the License for the
    specific language governing permissions and limitations
    under the License.

> airflowctl 1.0.0b1 (2025-08-19)

Reviewer: 23?

Author: Make sense! I plan to complete the document changes and have a staging version, hopefully today :crossed_fingers: . I will resync the staging after this one is merged if this is merged later than resync.

Author: Updated as 23

> airflowctl 1.0.0b1 (2025-08-23)

Author: ```suggestion
airflowctl 1.0.0b1 (2025-08-24)
```
Should we need to update this?

> Airflow CTL 1.0.0b1 (2025-08-19)

Reviewer 2: ```suggestion
`airflowctl` 1.0.0b1 (2025-08-19)
```

imo: we should just call it `airflowctl` like `kubectl` does:

https://kubernetes.io/docs/reference/kubectl/
https://kubernetes.io/docs/reference/kubectl/docker-cli-to-kubectl/

Author: Sure, I love the idea :) I have updated all occurrences

------------------------------

> Beta Release of Airflow CTL, a command-line tool. There are lots of great features to use from start.

Reviewer 2: ```suggestion
Beta Release of `airflowctl`, a command-line tool. There are lots of great features to use from start.
```
